src/ui/molecules/networkGraph/ecs/util.ts

```diff
@@ -0,0 +1,7 @@
+import { Vector3 } from '@babylonjs/core'
+import type { Vector } from 'ngraph.forcelayout'
+
+// eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+export const vector2Vector3: (pos: Vector) => Vector3 = (pos: Vector) =>
```

Comment:
I can't find where it is used ... Does it have a special behavior?

Reply:
Not used anymore. But could be used to bridge between the vectors from `ngraph.forcelayout` and `babylonjs`.